installerfw.pri

```diff
@@ -140,3 +140,19 @@ equals(TEMPLATE, app) {
     win32-g++*:POST_TARGETDEPS += $$IFW_LIB_PATH/libinstaller.a $$IFW_LIB_PATH/lib7z.a
     unix:POST_TARGETDEPS += $$IFW_LIB_PATH/libinstaller.a $$IFW_LIB_PATH/lib7z.a
 }
+
+# Protobuf
+# LIBS += -L$$PWD/src/libs/protobuf/lib/ -llibprotobuf
+# INCLUDEPATH += $$PWD/src/libs/protobuf/include
+PROTOBUF_PATH = /eve_installer/protobuf/v3.6.0
+LIBS += -L$$PROTOBUF_PATH/lib/ -llibprotobuf
+INCLUDEPATH += $$PROTOBUF_PATH/include
+
+# Platform detection module (and wrapper)
+# LIBS += -L$$PWD/src/libs/pdm/libs/Windows/x32/MT/ -lpdm -lplatform_pdm_proto_wrapper
+# INCLUDEPATH += $$PWD/src/libs/pdm/include
+# INCLUDEPATH += $$PWD/src/libs/pdm/include/generated
+PDM_PATH = /eve_installer/pdm-proto/34
```

Comment:
It is my way of versioning the pdm releases, since they don't do it themselves. This way updating to another version of the pdm is an explicit decision. And also gives us the ability to have more than one version of the pdm on our machines.

Reply:
But now that I think about it, it really only makes sense if you for some reason need to have two versions of the installers on your machine simultaneously, that use different versions of the pdm. Which is quite the edge case. I'll remove the 34 folder, and we'll just deal with the edge case if it ever comes up.